packages/strapi-plugin-content-manager/admin/src/components/Pagination/index.js

```diff
@@ -4,131 +4,65 @@
  *
  */
 
-import React from 'react';
+import { map } from 'lodash';
+import PropTypes from 'prop-types';
 
 import styles from './styles.scss';
 
 class Pagination extends React.Component {
-  constructor(props) {
-    super(props);
-    this.onGoPreviousPageClicked = this.onGoPreviousPageClicked.bind(this);
-    this.onGoNextPageClicked = this.onGoNextPageClicked.bind(this);
-    this.onGoFirstPageClicked = this.onGoFirstPageClicked.bind(this);
-    this.onGoLastPageClicked = this.onGoLastPageClicked.bind(this);
+  getLastPageNumber = () => {
+    return Math.ceil(this.props.count / this.props.limit);
   }
-  /**
-   * Triggered on dots click
-   *
-   * Simply prevent link default behavior
-   *
-   * @param e {Object} Click event
-   */
-  onDotsClicked(e) {
+
```

Comment:
Why did you remove the comments?

Reply:
Because I thought that the function naming was good enough to understand what the function does

Reply:
@soupette ?

Reply:
Well since the naming is explicit and the code within it is easy to understand I thought that the comments were useless...

Reply:
Comments can be useful to hep new comers understand what is the logic and the choices behind the functions.